migration/migration.go

```diff
@@ -427,28 +430,23 @@ func createOrUpdateWorkItemLinkCategory(ctx context.Context, linkCatRepo *link.G
 }
 
 func createOrUpdateSpace(ctx context.Context, spaceRepo *space.GormRepository, id uuid.UUID, description string) error {
-	spa, err := spaceRepo.Load(ctx, id)
+	_, err := spaceRepo.Load(ctx, id)
 	cause := errs.Cause(err)
-	space := &space.Space{
+	newSpace := &space.Space{
 		Description: description,
 		Name:        "system.space",
 		ID:          id,
 	}
 	switch cause.(type) {
 	case errors.NotFoundError:
-		_, err := spaceRepo.Create(ctx, space)
-		if err != nil {
-			return errs.WithStack(err)
-		}
-	case nil:
 		log.Info(ctx, map[string]interface{}{
 			"pkg":     "migration",
 			"spaceID": id,
-		}, "space %s exists, will update/overwrite the description", id)
-
-		spa.Description = description
-		_, err = spaceRepo.Save(ctx, spa)
-		return errs.WithStack(err)
+		}, "space %s will be created", id)
+		_, err := spaceRepo.Create(ctx, newSpace)
```

Comment:
Yes, that is a part I wanted to discuss about it. I found `version conflicts` so I tried a solution that updated the description + increased the version number. Problem I was having collisions (that is scary) cause multiple tests were trying to save an existing version number. Ofc, locking could solve it but I didn't see it in place here.